test-data/unit/stubgen.test

```diff
@@ -4342,3 +4342,19 @@ alias = tuple[()]
 def f(x: tuple[()]): ...
 
 class C(tuple[()]): ...
+
+[case testPreserveEnumValue_semanal]
```

Comment:
Can you test what happens with an enum over a nontrivial type, like a user-defined class?